### DISCUSSION TITLE
Robyn Tutorial

### DISCUSSION THREAD

**Discussion by argonne-cbs** [sparckles/Robyn#726] [Q&A] [answered]
Hi there. Been trying to follow the tutorial but I get a lot of errors that I cant get pass. Is the tutorial code anywhere on Github? also any examples with templates?

**Comment by sansyrox** [comment]
Hey @argonne-cbs 👋 

Thank you for opening the issue. Can you please point out which step is giving you errors?

You can find the final source code at - https://github.com/sparckles/example_robyn_app

**Reply by argonne-cbs** [reply]
Thanks, let me compare the code. I am pretty sure it is on my end of things. Will post back. Thanks

**Reply by argonne-cbs** [reply]
FYI the link is broken

**Reply by sansyrox** [reply]
@argonne-cbs , my bad. The repo was set to private for some reason. The link should work now 😄

**Reply by argonne-cbs** [reply]
Thanks!

**Reply by argonne-cbs** [reply]
Ok. So the code is correct. Maybe I am not starting the program the right way. Is it python3 crud.py to get the server started?

**Reply by sansyrox** [reply]
Hey @argonne-cbs 👋 

There was one bug in the code, I have fixed that. I have also added usage instructions in the example app README 😄 

It should be `python -m src` in this particular case

**Reply by argonne-cbs** [reply]
Thanks. I have been trying Python Frameworks trying to stick to one. I really like your tutorial example. It is easy to follow and covers a lot. To me, documentation/examples are so important especially for a novice like myself. Thanks for this!

**Reply by sansyrox** [reply, marked as answer]
Not a problem @argonne-cbs 😄 I am glad you like this style! Should I mark this discussion as resolved?
